File: problems/1832 - Check if the Sentence Is Pangram/PHP/Solution.php
<?php

class Solution {

    /**
     * @param String $sentence
     * @return Boolean
     */
    function checkIfPangram($sentence) {
        $chars = str_split($sentence);
        $alpha = range('a', 'z');
        $res = array_diff($alpha, $chars);
        
        return !$res;
    }
}

File: problems/1832 - Check if the Sentence Is Pangram/Go/Solution.go
func checkIfPangram(sentence string) bool {
    if len(sentence) < 26 {
        return false 
    }
    
    alpha := make(map[rune]bool)
    for _, c := range sentence {
        if alpha[c] == false {
            alpha[c] = true  
        }
          
        if (len(alpha) == 26) {
            return true; 
        }
    }
    return false;
}